doc/UserGuide.md

```diff
@@ -53,6 +53,10 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Listing all persons in sorted order : `sort`
+Shows a list of all persons in the address book in sorted order.<br>
+Format: `list`
+
```

Comment:
Good to see that you have updated the user guide :)

However, please make sure that the guide is updated correctly. There is something wrong with your update here (hint: under format)...